js/authentication.md

```diff
@@ -957,6 +957,7 @@ function refreshToken() {
         }
         res(data);
     });
+  }
```

Comment:
can you remove the trailing whitespace here? Happy to approve your PR with that minor change.